### PR TITLE
Jovatn/test spec minor fixes

### PR DESCRIPTION
**Pull request by jovatn**
## Description

Minor fixes to test descriptions and test steps, typically spelling errors, missing words, incorrect IP addresses, etc.

## Checklist

Tick *relevant* boxes, this PR is-a or has-a:

- [ ] Bugfix
  - [ ] Regression tests
  - [ ] ChangeLog updates (for next release)
- [ ] Feature
  - [ ] YANG model change => revision updated?
  - [ ] Regression tests added?
  - [ ] ChangeLog updates (for next release)
  - [ ] Documentation added?
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (please detail in commit messages)
- [ ] Build related changes
- [X] Documentation content changes
- [X] Other (please describe): Updates test documentation

### CODE REVIEW

File: test/case/ietf_hardware/usb/test.py [hunk 1]
```diff
@@ -2,13 +2,13 @@
...
 USB configuration
 
-This will test if the configuration is consistent with hardware state,
-if the USB ports are _locked_ (forbidden for use) and _unlocked_
-(Ready for use) when they should. Also test if it is the case during
-reboot. This does not the test actual use of the USB port, just check
+This tests if the configuration is consistent with hardware state,
+and if the USB ports are _locked_ (forbidden for use) and _unlocked_
+(Ready for use) when they should. Also tests if this is the case during
+reboot. This does not the test actual use of the USB port, it just checks
```
Comment (axkar) [inline test/case/ietf_hardware/usb/test.py:8]: Typo: "This does not **test the** actual use of ...."

Comment (axkar) [reply test/case/ietf_hardware/usb/test.py:8]: I can also suggest "(restricted from use)" and "(available for use)"

File: test/case/ietf_interfaces/verify_all_interface_types/test.py [hunk 2]
```diff
@@ -2,8 +2,8 @@
 r"""
 Verify that all interface types can be created
 
-This test verify that all interface types can be created
-and also, tesing setting the configuration in sequal (this
+This test verifies that all interface types can be created
+and also, testing setting the configuration in sequal (this
```
Comment (axkar) [inline test/case/ietf_interfaces/verify_all_interface_types/test.py:6]: Nothing major. Whatever you like more:
"This test verifies that all interface types can be created and also checks the configuration when applied sequentially. This method takes slightly longer than sending the entire configuration at once."

File: test/case/ietf_interfaces/dual_bridge/test.py [hunk 3]
```diff
@@ -7,7 +7,7 @@
 """
 Dual bridges
 
-Test that it is possible to ping a bridge through a bridge to another bridge with VETH interfaces.
+Test that it is possible to ping through a bridge to another bridge via VETH interfaces.
```
Comment (axkar) [inline test/case/ietf_interfaces/dual_bridge/test.py:10]: I prefer "Verify" over "Test", but pick what you like :)

Comment (axkar) [reply test/case/ietf_interfaces/dual_bridge/test.py:10]: Maybe something like:
"Verify that it is possible to ping through one bridge to another bridge using VETH interfaces"

File: test/case/ietf_routing/ospf_multiarea/Readme.adoc [hunk 4]
```diff
@@ -1,10 +1,10 @@
 === OSPF with multiple areas
 ==== Description
-This test test alot of features inside OSPF using 3 areas (one NSSA area, with no summary)
+This test tests a lot of features inside OSPF using 3 areas (one NSSA area, with no summary)
```
Comment (axkar) [inline test/case/ietf_routing/ospf_multiarea/Readme.adoc:3]: Maybe we could rephrase this a little bit if you like :
"This test evaluates various OSPF features across three areas (one NSSA area without summary) to ensure that route distribution is deterministic (based on cost). It also tests link failures using BFD, though BFD is not yet implemented in Infamy."

File: test/case/infix_services/services_basic/test.py [hunk 5]
```diff
@@ -3,7 +3,7 @@
 Services basic
 
 Verify that basic services like mDNS and LLDP can be enabled and
```
Comment (axkar) [inline test/case/infix_services/services_basic/test.py:5]: Very minor suggestion, if you like:
"Verify that basic services, such as mDNS and LLDP, can be enabled and disabled. Operation and non-operation are confirmed using tcpdump."

File: test/case/ietf_hardware/usb/test.py (hunk 1 above)
Comment (axkar) [reply test/case/ietf_hardware/usb/test.py:8]: I can suggest an overall change, but feel free to adopt whatever piece suits you:
"This test checks if the configuration aligns with the hardware state and verifies whether the USB ports are correctly locked (restricted from use) and unlocked (available for use) when they should. It also verifies this behavior during a reboot. The test does not involve the actual use of the USB ports; it only ensures that the configured state matches the hardware state."

**Review by axkar — APPROVED** [review]

File: test/case/ietf_hardware/usb/test.py (hunk 1 above)
Comment (jovatn) [reply test/case/ietf_hardware/usb/test.py:8]: Thanks. I used most of the comments.

File: test/case/ietf_interfaces/verify_all_interface_types/test.py (hunk 2 above)
Comment (jovatn) [reply test/case/ietf_interfaces/verify_all_interface_types/test.py:6]: I agree this is an improvement, and use it as is. (I also think we can remove the last sentence, but I postpone that for now.)

File: test/case/ietf_interfaces/dual_bridge/test.py (hunk 3 above)
Comment (jovatn) [reply test/case/ietf_interfaces/dual_bridge/test.py:10]: Fixed

File: test/case/ietf_routing/ospf_multiarea/Readme.adoc (hunk 4 above)
Comment (jovatn) [reply test/case/ietf_routing/ospf_multiarea/Readme.adoc:3]: I'll use most of your suggestions. Thx!

**Pull request merged**